userspace/libsinsp/parsers.cpp

```diff
@@ -601,7 +601,16 @@ bool sinsp_parser::reset(sinsp_evt *evt)
 		query_os = true;
 	}
 
-	evt->m_tinfo = m_inspector->get_thread(evt->m_pevt->tid, query_os, false);
+	evt->m_tinfo_ref.reset();
```

Comment:
Can you move this to sinsp_evt::init? (All 3 versions, sadly).

Reply:
sure, no problem